### PR TITLE
Add support for peersrpc service

### PR DESCRIPTION
**Pull request by orbitalturtle**
This adds support for Lightning's peersrpc service. This will be useful for the [lndk project](https://github.com/carlaKC/lndk).

I used [this PR](https://github.com/Kixunil/tonic_lnd/pull/21) as a reference, since it makes much the same additions but for the walletkit service.

Thanks much to anyone able to review. :]

### CODE REVIEW

**Comment by Kixunil** [comment]
Needs rebase.

**Review by carlaKC — APPROVED** [review]
Looks good after rebase!

File: vendor/peersrpc/peers.proto [hunk 1]
```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+
+import "lightning.proto";
+
+package peersrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/peersrpc";
+
+// Peers is a service that can be used to get information and interact
+// with the other nodes of the network.
```
Comment (Kixunil) [inline vendor/peersrpc/peers.proto:10]: FYI this line differs from the original but I'll allow it.

**Review by Kixunil — APPROVED** [review]

**Comment by Kixunil** [comment]
Thanks for the PR! Do you need a release soonish?

**Comment by Kixunil** [comment]
Will try to release in at most a few days.

**Comment by orbitalturtle** [comment]
> Thanks for the PR! Do you need a release soonish?

Thanks for the review!

**Comment by Kixunil** [comment]
Released!

**Pull request merged**